pkg/api-server/inspect_endpoints_test.go

```diff
@@ -1236,6 +1238,82 @@ var _ = Describe("Inspect WS", func() {
 			},
 			contentType: "text/plain",
 		}),
+		Entry("inspect rules", testCase{
+			path:    "/meshes/default/dataplanes/backend-1/rules",
+			matcher: matchers.MatchGoldenJSON(path.Join("testdata", "inspect_dataplane_rules.json")),
```

Comment:
Can we have golden in the filename (it helps understand tests during review)

Reply:
Good point. Other files didn't have this so I forgot to add it. Added